src/order/hom/basic.lean

```diff
@@ -623,6 +623,16 @@ have gf : ∀ (a : α), a = g (f a) := by { intro, rw [←cmp_eq_eq_iff, h, cmp_
   right_inv := by { intro, rw [←cmp_eq_eq_iff, ←h, cmp_self_eq_eq] },
   map_rel_iff' := by { intros, apply le_iff_le_of_cmp_eq_cmp, convert (h _ _).symm, apply gf } }
 
+/-- To show that `f : α →o β` and `g : β →o α` make up an order isomorphism it is enough to show
+    that `g` is the inverse of `f`-/
+def of_hom_inv {f : α →o β} {g : β →o α} (h₁ : function.left_inverse g f)
```

Comment:
@eric-wieser told me that these are usually stated in terms of `.comp`; for example [`alg_equiv.of_alg_hom`](https://leanprover-community.github.io/mathlib_docs/algebra/algebra/basic.html#alg_equiv.of_alg_hom), [`ring_equiv.of_hom_inv`](https://leanprover-community.github.io/mathlib_docs/algebra/ring/equiv.html#ring_equiv.of_hom_inv). maybe this is the right way and you change the `left_inverse` stuff below too? (both should be fairly easy from each other)

Reply:
(this goes with his point about the left_inv+right_inv stuff)